.eslintrc.js

```diff
@@ -19,9 +19,21 @@ module.exports = {
       ImportDeclaration: { multiline: true, minProperties: 6 },
       ExportDeclaration: { multiline: true, minProperties: 6 },
     }],
+    'no-unused-expressions': 0,
+    'chai-friendly/no-unused-expressions': 2,
```

Comment:
@vhargrave, the change does not only remove the generic rule, it replaces it with a "chai friendly" one.